README.md

```diff
@@ -104,6 +104,10 @@ Run the schema generator via `node main.js`.
 
 -o, --out
     Specify the output file path. Without this option, the generator logs the response in the console.
+
+--allow-additional-properties  <true|false>
+    true: Allow objects to have additional properties
+    false (default): Do not allow objects to have additional properties
```

Comment:
That's not entirely true. We still allow additional properties but you have to write something like

```ts
{
  [key: string]: number
}
```

Can you rewrite the description to be more precise?

Reply:
How about this?

```
--additional-properties  <true|false>
    Controls whether or not to allow additional properties for objects that have no index signature.

    true: Additional properties are allow
    false (default): Additional properties are not allowed
```

Reply:
Love it. Thanks!

Additional properties are allow -> Additional properties are allowed